marblerun/reference/coordinator.md

```diff
@@ -139,9 +135,9 @@ This API endpoint only works when `Users` were defined in the manifest. For more
 | \<SecretName\> (one or more) | map  | A map containing key-value pairs for the requested secret. |
 
 
-Each GET requests allows specifying one or more secrets in the form of a query string, where each parameter `s` specifies one secret.
-A query string for the secrets `symmetric_key_shared` and `cert_shared` may look like the following:
-```
+Each GET request allows specifying one or more secrets in the form of a query string, where each parameter `s` specifies one secret.
+A query string for the secrets `symmetric_key_shared` and `cert_shared` may look like this:
+```php
```

Comment:
```suggestion
```

We do not use a php linter in the docs.

Reply:
I used it to highlight the request parameters, but if this is not uspported we can leave it out.